.github/workflows/create_rc_pr.yml

```diff
@@ -87,17 +96,18 @@ jobs:
                 WARNING: ${{ needs.find_release_branches.outputs.warning }}
               run: |
                 if [ -n "${{ needs.find_release_branches.outputs.warning }}" ]; then
-                  echo "CHANGES=$(inv -e release.check-for-changes -r "$MATRIX" "$WARNING")" >> $GITHUB_OUTPUT
+                  echo "CHANGES=$(inv -e release.check-for-changes -r "$MATRIX" "$WARNING" | tail -n 1)" >> $GITHUB_OUTPUT
                 else
-                  echo "CHANGES=$(inv -e release.check-for-changes -r "$MATRIX")" >> $GITHUB_OUTPUT
+                  echo "CHANGES=$(inv -e release.check-for-changes -r "$MATRIX" | tail -n 1)" >> $GITHUB_OUTPUT
                 fi
 
             - name: Create RC PR
-              if: ${{ steps.check_for_changes.outputs.CHANGES == 'true'}}
+              if: ${{ steps.check_for_changes.outputs.CHANGES == 'true' || env.IS_AGENT6_RELEASE == 'true' }}
```

Comment:
Should we remove the previous step if the output is ignored when releasing agent 6 ?

Reply:
No, I think we should keep it because the `check_for_changes` function not only checks for changes but can also modifies the tags. There is additional logic that needs to be executed as part of the process [here](https://github.com/DataDog/datadog-agent/blob/main/tasks/release.py#L1214-L1219).